schema.sql

```diff
@@ -16,4 +16,18 @@ ALTER TABLE animals ADD CONSTRAINT fk_species_id FOREIGN KEY (species_id) REFERE
 ALTER TABLE animals ADD COLUMN owner_id INT;
 ALTER TABLE animals ADD CONSTRAINT fk_owner_id FOREIGN KEY (owner_id) REFERENCES owners (id);
 
+CREATE TABLE vets (id INT GENERATED ALWAYS AS IDENTITY, name VARCHAR(250), age INT, date_of_graduation DATE, PRIMARY KEY(id));
+
+CREATE TABLE spevialization (species_id INT, vets_id INT, PRIMARY KEY (species_id, vets_id),
+  CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id),
+  CONSTRAINT fk_vets FOREIGN KEY(vets_id) REFERENCES vets(id) 
+);
+CREATE TABLE visits( animal_id INT, vets_id INT,visiting_date DATE,
+  PRIMARY KEY (animal_id, vets_id, visiting_date),
+  CONSTRAINT fk_animals FOREIGN KEY(animal_id) REFERENCES animals(id),
+  CONSTRAINT fk_vets FOREIGN KEY(vets_id) REFERENCES vets(id)
+);
```

Comment:
- [ ] Good job so far :clap:  But when I tried to create the visit table I'm getting the errors below :cry: Please make sure all your table structures are well designed in order to make your database performance great :smile: kindly fix the error :+1: 
![Screenshot from 2022-10-20 23-04-11](https://user-images.githubusercontent.com/6615394/197076234-2033d0e4-506c-448e-b686-473fc2c47ffe.png)

Reply:
The primary key was added by ALTER query for animals. now I include it on the create animals query and this error is resolved

Reply:
Good job !